warehouse/dbt_project.yml

```diff
@@ -92,3 +92,5 @@ models:
         schema: mart_payments
       benefits:
         schema: mart_benefits
+      transit_data_quality_issues:
```

Comment:
Or maybe `mart_transit_database`... Since this will have foreign keys to other tables there and `transit database` is kind of just an alias for `Airtable`.

Reply:
Resolved in [d43974a](https://github.com/cal-itp/data-infra/pull/2781/commits/d43974a56418e258bfecc83cbff34011e70b5dbc)